### DISCUSSION TITLE
No an issue, can i use this with my own model?

### DISCUSSION THREAD

**Discussion by ghost** [lkwq007/stablediffusion-infinity#44] [Q&A]
In other colabs I only have to load my model but I'm not sure how to do it here

**Comment by lkwq007** [comment]
Check out https://github.com/lkwq007/stablediffusion-infinity/issues/34

Also note that this project only supports the model weights downloaded from the huggingface hub

**Comment by toonpeters** [comment]
updates?

**Comment by lekjaz** [comment]
I am no programmer, but I have had good luck using multiple models, as long as they are valid checkpoint files. Just provide the filepath to where the model you want to use is located and it will use it.
